src/utils/webrtc/index.js

```diff
@@ -333,24 +333,28 @@ async function signalingIsConnected(signaling) {
 
 	const signalingConnectionSucceededOnConnect = () => {
 		signaling.off('connect', signalingConnectionSucceededOnConnect)
-		signaling.off('error', signalingConnectionFailedOnInvalidToken)
+		signaling.off('error', signalingConnectionFailedOnError)
 
 		signalingConnectionSucceeded()
 	}
 
-	const signalingConnectionFailedOnInvalidToken = (error) => {
-		if (error.code !== 'invalid_token') {
+	const signalingConnectionFailedOnError = (error) => {
+		if (error.code !== 'invalid_token' && error.code !== 'invalid_client_type') {
```

Comment:
Personal preference - looks more readable to me, I see that error.code should be one of these
```suggestion
		if (!['invalid_token', 'invalid_client_type'].includes(error.code)) {
```

Reply:
I have to say, I would also prefer the format of the PR in this case (so with 2 `!==` cases).

Sorry @Antreesy 🥲